packages/superset-ui-legacy-preset-chart-nvd3/src/LineMulti/LineMulti.jsx

```diff
@@ -58,16 +61,27 @@ function getJson(url) {
 class LineMulti extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { queryData: [] };
+    this.state = { queryData: false };
   }
 
   componentDidMount() {
     this.loadData(this.props);
   }
 
-  // eslint-disable-next-line react/no-deprecated
-  componentWillReceiveProps(nextProps) {
-    this.loadData(nextProps);
+  shouldComponentUpdate(nextProps) {
+    const { rawFormData } = this.props;
+    const { rawFormData: nextRawFormData } = nextProps;
+    const { queryData } = this.state;
+    if (!queryData || !isEqual(rawFormData, nextRawFormData)) {
+      return true;
+    }
+
+    return false;
+    // return false;
```

Comment:
Please remove comment

Reply:
huh... wish I had a good explanation for that! It's gone now... let's see if it builds! :D